run-tests

```diff
@@ -1,7 +1,16 @@
-#!/bin/sh
+#!/bin/bash --
 
+set -eu
+unset applications do_install_deps CLEANUP_LVM name DEFAULT_LVM_POOL retcode
+applications=(lvm2 python3-inotify python3-sphinx python3-docutils
+    python3-pyyaml python3-jinja2 python3-lxml python3-pylint python3-coverage
+    btrfs-progs vim-common)
+do_install_deps=false
+rpm -q --quiet -- "${applications[@]}" ||
+sudo dnf -- install "${applications[@]}"
```

Comment:
I'd disarm it even further, to not break when running on Debian...

Reply:
Fixed by checking that `rpm` and `dnf` are installed in the standard search path before using them.